interpreter/text/annot.ml

```diff
@@ -22,8 +22,14 @@ module NameMap = Map.Make(struct type t = Ast.name let compare = compare end)
 type map = annot list NameMap.t
 
 let current : map ref = ref NameMap.empty
+let current_source : Buffer.t = Buffer.create 512
 
-let clear () = current := NameMap.empty
+let reset () =
+  current := NameMap.empty;
+  Buffer.clear current_source
+
+let get_source () =
+  Buffer.sub current_source 0 (Parsing.symbol_end ())
```

Comment:
Why not simply `Buffer.contents current_source`?

Reply:
Well, but is it ensured that it _starts_ with the current module? I believe there could be other stuff before it as well, so it's perhaps a bit odd to treat the end differently.

Reply:
that's a good point.
Originally I was using `Parsing.symbol_end()` for the start of the range, but then I realized that in that case all the source positions will be skewed and useless.

Reply:
I wil just return the whole contents then. We can revisit this in the future if it will actually improve parsing for custom handlers